### PR TITLE
remove Tech preview of libssh

### PR DESCRIPTION
**Pull request by akira6592**
##### SUMMARY

I heard that libssh is not Tech preview now in ansible network slack workspace.
So I removed `Tech preview` of libssh,

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

libssh connection plugin document

### CODE REVIEW

**Comment by Qalthos** [comment]
recheck

**Comment by akira6592** [comment]
@Qalthos In this case, should I create a changelog fragment?

**Comment by Qalthos** [comment]
> @Qalthos In this case, should I create a changelog fragment?

Yes, you should. I would probably put this under `minor_changes` as although nothing has really changed, the change in status is still worth calling out in the changelog

**Comment by akira6592** [comment]
@Qalthos Thank you. I added changelog fragment as `minor_changes`.

File: changelogs/fragments/451-libssh-remove-tech-preview.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```
Comment (Qalthos) [inline changelogs/fragments/451-libssh-remove-tech-preview.yaml:2]:
```suggestion
bugfixes:
```
Sorry for waffling on this when this is what I told you to do earlier. I don't think there's really a good section to put this under but it does need to get called out. Docs bugfixes are still bugfixes though.

**Review by Qalthos — CHANGES_REQUESTED** [review]

**Review by Qalthos — APPROVED** [review]

**Pull request merged**